src/commands/initProjectForVSCode/InitVSCodeStep/InitVSCodeStepBase.ts

```diff
@@ -152,7 +152,10 @@ export abstract class InitVSCodeStepBase extends AzureWizardExecuteStep<IProject
     private getTaskLabel(task: ITask): string | undefined {
         switch (task.type) {
             case func:
-                return `${task.type}: ${task.command}`;
+                if (!task.label) {
+                    return task.type
+                }
+            // eslint-disable-next-line no-fallthrough
```

Comment:
Rather than suppress this, I'd add an `else` to retain clarity about what should happen.